### PR TITLE
fix(@ngtools/webpack): fix Angular 5 lazy routes in es2015

### LINKED ISSUE #7689: warning raised if targeting es2015
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.5.0-beta.0
node: 8.5.0
os: linux x64
@angular/animations: 5.0.0-beta.7
@angular/common: 5.0.0-beta.7
@angular/compiler: 5.0.0-beta.7
@angular/core: 5.0.0-beta.7
@angular/forms: 5.0.0-beta.7
@angular/http: 5.0.0-beta.7
@angular/platform-browser: 5.0.0-beta.7
@angular/platform-browser-dynamic: 5.0.0-beta.7
@angular/router: 5.0.0-beta.7
@angular/cli: 1.5.0-beta.0
@angular/compiler-cli: 5.0.0-beta.7
@angular/language-service: 5.0.0-beta.7
typescript: 2.5.2

### Repro steps.
ng new demo
update target in tsconfig.json to 'es2015'
ng build

### The log given by the failure.
```
WARNING in ./node_modules/@angular/core/esm15/index.js
5132:15-36 Critical dependency: the request of a dependency is an expression
    at ImportLazyContextDependency.getWarnings (/home/sander/Documenten/test/demo/node_modules/webpack/lib/dependencies/ImportContextDependency.js:28:4)
    at Compilation.reportDependencyErrorsAndWarnings (/home/sander/Documenten/test/demo/node_modules/webpack/lib/Compilation.js:694:24)
    at Compilation.finish (/home/sander/Documenten/test/demo/node_modules/webpack/lib/Compilation.js:552:9)
    at applyPluginsParallel.err (/home/sander/Documenten/test/demo/node_modules/webpack/lib/Compiler.js:512:17)
    at /home/sander/Documenten/test/demo/node_modules/tapable/lib/Tapable.js:289:11
    at _addModuleChain (/home/sander/Documenten/test/demo/node_modules/webpack/lib/Compilation.js:498:11)
    at processModuleDependencies.err (/home/sander/Documenten/test/demo/node_modules/webpack/lib/Compilation.js:468:14)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```
### Desired functionality.
clear the warning, it's pushing essential information out of the view!

### PR DESCRIPTION
**Pull request by filipesilva**
Fix #7689

### CODE REVIEW

File: packages/@ngtools/webpack/src/plugin.ts [hunk 1]
```diff
@@ -358,6 +358,14 @@ export class AotPlugin implements Tapable {
       // being built.
```
Comment (hansl) [inline packages/@ngtools/webpack/src/plugin.ts]: Do we need the changes here?  This is only for Angular 5, isn't it?

**Comment by hansl** [comment]
The commit is not about fixing the warning, it's about fixing the resolution of lazy routes in Angular 5, isn't it?

File: packages/@ngtools/webpack/src/plugin.ts (hunk 1 above)
Comment (filipesilva) [reply packages/@ngtools/webpack/src/plugin.ts]: It's still possible to turn off the new compiler (`--no-experimental-angular-compiler`) so this plugin should still work with Angular 5.

**Comment by filipesilva** [comment]
Fixed commit message.

**Comment by petebacondarwin** [comment]
where are the tests? :-P

**Comment by hansl** [comment]
@petebacondarwin we test against Angular 5.0.0 already, with a lazy route :-)

**Comment by filipesilva** [comment]
@hansl we run this test in nightly, and it tests lazy routes https://github.com/angular/angular-cli/blob/master/tests/e2e/tests/build/rebuild.ts.

**Comment by petebacondarwin** [comment]
@filipesilva - If you already have tests for this then why were they not failing already? Or perhaps they were, which is what led to this PR?

**Comment by filipesilva** [comment]
@petebacondarwin because this only failed when lazy routes were being used with es2015. We test the two but separately.

**Pull request merged**